src/cases/containers/case-viewer-container/case-viewer-container.component.ts

```diff
@@ -18,7 +18,7 @@ import * as fromRoot from '../../../app/store';
 export class CaseViewerContainerComponent implements OnInit {
   private static readonly FEATURE_WORK_ALLOCATION_RELEASE_1 = 'WorkAllocationRelease1';
   private static readonly FEATURE_WORK_ALLOCATION_RELEASE_2 = 'WorkAllocationRelease2';
-  private workAllocationRoles: string[] = ['caseworker-ia-iacjudge'];
+  private workAllocationRoles: string[] = ['caseworker-ia-iacjudge', 'caseworker-ia'];
```

Comment:
I think caseworker-ia-caseofficer is for legal ops, don't use 'caseworker-ia' as solicitors also have this role.
I have defined the whitelist roles here for future when adding other jurisdiction's Judicial/LegalOps users.
import { JUDICIAL_ROLE_LIST, LEGAL_OPS_ROLE_LIST } from './app.constants';

Reply:
There has been a generic AppUtils.isLegalOpsOrJudicial(userRoles) to determine whether the user is LegalOps or Judicial users.